### ISSUE TITLE
Wrong color value for types

### ISSUE THREAD

**Issue by alevroub** [markusylisiurunen/github-dark-mode#12]
Love this theme. Though for consistency with the web editor, I think the color for types should be blue `#79c0ff` and not red `#ff7b72`. This is what I'm using for now:

```
"editor.tokenColorCustomizations": {
   "[GitHub Dark Mode]": {
      "types": "#79c0ff",
   }
}
```

**Comment by markusylisiurunen** [comment]
Hi! Could you give me some sample code where you find this inconsistency between the theme and GitHub's web editor?

**Comment by alevroub** [comment]
Of course. So far I've only seen this affect json property names:

<img width="1643" alt="Screenshot 2021-02-05 at 15 34 02" src="https://user-images.githubusercontent.com/6848068/107047051-acc9ef00-67c7-11eb-997b-efddb6727791.png">

**Comment by markusylisiurunen** [comment]
Thanks! I added a fix for JSON field name colors in #13 and an update should be available in a few minutes. If you find other inconsistencies, feel free to report them as separate issues 🙂
